rickshaw-post-process-bench

```diff
@@ -330,5 +325,10 @@ for (my $i = 1; $i <= scalar @{ $run{'iterations'} }; $i++) {
             }
             print "Post-processing complete\n";
         }
+        if (scalar @samp_dirs < $run{'num-samples'}) {
+            printf "Iteration %d, [%s] is considered failed because there were not enough passing samples (%d)\n",
+                   $i, $this_iter_dir, $run{'num-samples'};
+            system("/bin/mv " . $run_dir . "/" . $this_iter_dir . " " . $run_dir . "/" . $this_iter_dir . "-fail");
+        }
```

Comment:
Is this change to ensure that the post-processing runs before the directory is rename?  Something else?

Reply:
yes